.github/workflows/update-sdk.yml

```
@@ -0,0 +1,111 @@
+name: Update SDK
+
+on:
+  schedule:
+    - cron: "0 14 * * 2"
+  workflow_dispatch:
+
+jobs:
+  update-sdk:
+    name: Update SDK if Necessary
+    runs-on: ${{ fromJSON(vars.SWI_GLORG_UBUNTU_2204) }}
```

```suggestion
    runs-on: ${{ fromJSON(vars.SWI_GLORG_UBUNTU_2204) }}
```
Shared runners don't work on public repos.

---

@ajrice6713 runs are going to still blow up on you.